Assets/Scripts/Enums/BodyPartType.cs

```diff
@@ -0,0 +1,18 @@
+namespace Enums
+{
+    public enum BodyPartType
```

Comment:
Maybe add a comment that his is for chopping off limbs, as it is missing some types for targeting.
Also add an explicit integer to each, such that removing or rearranging the enums doesn't affect anything in Unity.